packages/ui-tickets/src/comment/components/Comment.tsx

```
@@ -52,65 +52,65 @@ class Comment extends React.Component<Props, State> {
+                (e.target as HTMLImageElement).src = "/images/avatar-colored.svg";
+              }}
+            />
+            <div>
+              <CommentContent>
+                <h5>
+                  {createdUser.fullName ||
+                    `${createdUser.firstName || ''} ${createdUser.lastName || ''}`}
+                </h5>
+                <div
+                  className="comment"
+                  dangerouslySetInnerHTML={{ __html: comment.content }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)